content/1.documentation/2.clients/1.cli.md

```diff
@@ -49,16 +49,40 @@ hopp test -e environment.json kitchen-sink-hoppscotch-collection.json
 
 Hoppscotch allows templates in several places. For example, you could specify your endpoint URL as `<<baseurl>>/post` and specify baseurl as [`https://echo.hoppscotch.io`](https://echo.hoppscotch.io) in an environment file.
 
-The expected format of an environment file is:
+Hoppscotch CLI supports environment files in two specific formats:
+
+### 1. Multiple Environment Variables Export Format
+
+This format is generated by Hoppscotch when you export any of your environment. It includes a named environment (nameOfEnv) with multiple key-value pairs, allowing you to define various variables within a single file.
+
+  ```json
+  {
+    "nameOfEnv": "my_env",
+    "variables": [
+      {
+        "key1": "base_url",
+        "value1": "https://echo.hoppscotch.io"
+      },
+      {
+        "key2": "auth_token",
+        "value2": "xxxxxxxxxxxx"
+      }
+    ]
+  }
+```
+
+### 2. Single Environment Variable Export Format
```

Comment:
The name suggests that we are using the single export entry format which is mentioned in point 1. It is better to rename this to something like a legacy format or another apt name.